local_groups/forms.py

```diff
@@ -152,8 +152,8 @@ class GroupLoginForm(AuthenticationForm):
         widget=forms.TextInput(attrs={'autofocus': True}),
         help_text='''
             This should be the official group leader email address registered
-            with Our Revolution, or the email you use for submitting
-            nominations
+            with Our Revolution, a group admin email, or the email you use for
```

Comment:
Can we also change label to something more generic than `Group Admin Email`?

Reply:
👍

Reply:
@cjmabry thanks! I noticed this page also has the title/page title with `Group Admin Login`. Can we change that to be generic too?